frontend/src/AddNewParticipant.tsx

```diff
@@ -0,0 +1,36 @@
+import {FormEvent, useState} from "react";
+import Button from '@mui/material/Button';
+import SendIcon from '@mui/icons-material/Send';
+import './AddNewParticipant.css';
+import {toast} from "react-toastify";
+
+export default function AddNewParticipant(props:
+                                              {
+                                                  addParticipant: (name: string) => Promise<void>,
+                                              }) {
+
+    const [name, setName] = useState("");
+    const onNameSubmit = (event: FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        if (name === "") return;
```

Comment:
Als JS Profi reicht es wenn du auf falsy testst.
Zu dem wäre es schön wenn der Nutzer in diesem Fall eine Fehlermeldung bekommt.
```suggestion
        if (name) return;
```